mmhuman3d/models/body_models/star.py

```diff
@@ -0,0 +1,212 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+# Adapted from:
+#
+# https://github.com/ahmedosman/STAR/blob/master/star/pytorch/star.py
+#
+#
+# -*- coding: utf-8 -*-
+#
+# LICENSE:
```

Comment:
Let's move the additional license to this [file](https://github.com/open-mmlab/mmhuman3d/blob/main/docs/additional_licenses.md).